src/fn/initialise-project-layout.js

```diff
@@ -48,7 +48,7 @@ const LAYOUT = {
 function createRecursively(dir, layout) {
   fs.mkdir(dir);
 
-  for (const k in layout) {
+  for (const k of Object.keys(layout)) {
```

Comment:
eslint was complaining that the for-in was risky since you might end up iterating on things from the object's prototype.  Doing a for-of of the `Object.keys` gets us the same thing, but  without the warning...